apps/experiments/views/experiment.py

```diff
@@ -457,6 +469,43 @@ class DeleteFileFromExperiment(BaseDeleteFileView):
     pass
 
 
+# TODO: complete form
+class ExperimentVersionForm(forms.ModelForm):
+    class Meta:
+        model = Experiment
+        fields = ["description"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["description"].widget.attrs.update({"class": "input input-bordered", "style": "width: 50%"})
```

Comment:
We don't have a version description field yet. This will update the experiment's own description, which I'm guessing we don't want to do?

Reply:
hmm yes I was thinking that it would update the copied description filed that is then unique to that version-- but I'll remove for now and plan to add in later

Reply:
https://github.com/dimagi/open-chat-studio/commit/307d0fe2afa335313a4ca7bebea09f8ca19fd67f (apologies for the force push just amended a file I forgot to add to the last commit no other changes